src/components/transactions/TxDetails/Summary/index.tsx

```diff
@@ -73,6 +74,16 @@ const Summary = ({ txDetails, defaultExpanded = false, hideDecodedData = false }
         </TxDataRow>
       )}
 
+      <SafenetTxChecks
```

Comment:
From what I understand, components in `src/components/tx/SignOrExecuteForm/...` are for the signing and executing form itself.

I think it would make more sense to not use the `SafenetTxChecks` component here (it uses things like `TxCard` which are MUI cards for the sign-or-execute form), and instead just embed the `SafenetTxSimulation` component (which is where all the logic for displaying the checks is anyway). This way, it can be laid out differently in the transaction summary to better match the UI there.

Reply:
Agree, I'll make a couple of change to reflect this